components/brave_today/browser/direct_feed_controller.cc

```diff
@@ -77,13 +86,22 @@ mojom::ArticlePtr RustFeedItemToArticle(const FeedItem& rust_feed_item) {
 
 using ParseFeedCallback = base::OnceCallback<void(absl::optional<FeedData>)>;
 void ParseFeedDataOffMainThread(const GURL& feed_url,
-                                const std::string& body_content,
+                                const std::string& charset,
+                                std::string body_content,
```

Comment:
I guess we can change this to be `const &` while leaving lambda parameter as is.
```suggestion
                                const std::string& body_content,
```

Reply:
Okay, I'm kinda surprised it compiles with a const reference here, as I'm modifying the string in the `base::ConvertToUTF8AndNormalize` function. I feel like this maybe shouldn't be a const reference because we're modifying the variable (but it might just be my poor C++ shining through :laughing: )

Reply:
Yeah, I know. The magic here is that base::Bind() copies the given |charset| and bind it. Like you said, I agree that making this to const & could be misleading. It's up to you. NVM.